Modelica/Blocks/Continuous.mo

```diff
@@ -765,7 +765,7 @@ to compute u by an algebraic equation.
       "Control error (set point - measurement)";
     parameter .Modelica.Blocks.Types.SimpleController controllerType=
            .Modelica.Blocks.Types.SimpleController.PID "Type of controller";
-    parameter Real k(min=0) = 1 "Gain of controller";
+    parameter Real k(min=1) = 1 "Gain of controller";
```

Comment:
The conclusion was that the `min` should be removed, not changed.

Reply:
I removed it at first, but thought it would be better to have ` min` as 1, so the user does not provide a zero value. 

Reading through comments in the issue, looks like it is okay. I will remove the `min`.